### PR TITLE
Support exporting backing image from volume

### PR DESCRIPTION
**Pull request by shuo-wu**
longhorn/longhorn#2403

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -120,3 +130,99 @@ func ReadBackingImageConfigFile(workDirectory string) (*BackingImageConfig, erro
...
 	return cfg, nil
 }
+
+func Execute(envs []string, binary string, args ...string) (string, error) {
+	return ExecuteWithTimeout(time.Minute, envs, binary, args...)
+}
+
+func ExecuteWithTimeout(timeout time.Duration, envs []string, binary string, args ...string) (string, error) {
+	var err error
+	cmd := exec.Command(binary, args...)
```
Comment (joshimoo) [inline pkg/util/util.go:140]: You can use a CommandContext, then pass a context.withTimeout. This way you don't need the timeout go routine below.

**Review by joshimoo — APPROVED** [review]
LGTM

File: pkg/util/util.go (hunk 1 above)
Comment (shuo-wu) [reply pkg/util/util.go]: Addressed.

**Review by innobead — APPROVED** [review]
LGTM

**Pull request merged**